src-java/base-topology/base-storm-topology/src/test/java/org/openkilda/wfm/share/mappers/HistoryMapperTest.java

```diff
@@ -0,0 +1,159 @@
+/* Copyright 2022 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.wfm.share.mappers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.openkilda.messaging.payload.history.FlowDumpPayload;
+import org.openkilda.messaging.payload.history.FlowHistoryEntry;
+import org.openkilda.messaging.payload.history.FlowHistoryPayload;
+import org.openkilda.model.FlowEncapsulationType;
+import org.openkilda.model.PathComputationStrategy;
+import org.openkilda.model.SwitchId;
+import org.openkilda.model.history.FlowEvent;
+import org.openkilda.model.history.FlowEventAction;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.mapstruct.factory.Mappers;
+
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.util.ArrayList;
+
+public class HistoryMapperTest {
+    public static final HistoryMapper INSTANCE = Mappers.getMapper(HistoryMapper.class);
```

Comment:
Why not use HistoryMapper.INSTANCE instead?

Reply:
Yes, it would be better, thanks.